### ISSUE TITLE
Mobile: doesn´t start automatically at 360p

### ISSUE THREAD

**Issue by benhartwich** [kylegilman/video-embed-thumbnail-generator#310]
Hi,

when I watch a video on mobile phone (https://univideo.uni-passau.de/2016/03/gruendercafe-jodel-edition/) it starts at 720p instead of 360p - see attached.
![screenshot_2016-03-30-10-30-13](https://cloud.githubusercontent.com/assets/4061954/14136359/c995ca2c-f662-11e5-8fc8-76c738e32401.png)

What could be the reason for? I used an Oneplus 2.

**Comment by kylegilman** [comment]
On the plugin settings page, what is the "Default playback resolution" set to? If it's on automatic, this is not unusual behavior, since mobile devices have such high-density screens and in a responsive design the video takes up more of the screen than it does on a desktop browser.

**Comment by kylegilman** [comment]
I checked the link, and it is set to automatic. If you want the video to load 360p by default, I recommend changing the default playback resolution setting to "lowest"

**Comment by benhartwich** [comment]
I think a automatic mobile detection would be the best. Perhaps this js would be suitable for implementation? 
http://detectmobilebrowsers.com/
https://github.com/kaimallea/isMobile

**Comment by kylegilman** [comment]
I am not interested in JS mobile detection. I used it a few years ago and it is not a good option. I could change the way screen resolution is calculated to achieve the result you're looking for, but I don't see why being on a mobile device should make the resolution lower. That kind of thinking is what just got [Netflix into trouble](http://www.theverge.com/2016/3/24/11302446/netflix-admits-throttling-video-att-verizon-customers). The pixel density in relation to the size of the player on the screen are higher on mobile. That's why the resolution is set higher when using the automatic setting. If you want it to be set lower, I've provided that option too.
